MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/kits/KitRegistry.java

```diff
@@ -56,23 +68,219 @@ public void registerKit(Kit kit) {
+      if (!configurationSection.contains("required-level")) {
+        configurationSection.set("required-level", ((LevelKit) kit).getLevel());
+      }
+    }
+
+    AtomicInteger currentItem = new AtomicInteger();
+    ConfigurationSection inventoryConfigurationSection = configurationSection.getConfigurationSection("Inventory");
+    if (inventoryConfigurationSection == null) {
+      inventoryConfigurationSection = configurationSection.createSection("Inventory");
+      final ConfigurationSection finalInventoryConfigurationSection = inventoryConfigurationSection;
+
+      kit.getKitItems().forEach((item, slot) -> {
+        ConfigurationSection itemConfigurationSection = finalInventoryConfigurationSection.createSection(String.valueOf(currentItem.get()));
+
+        ConfigurationSection itemStackConfigurationSection = itemConfigurationSection.createSection("item");
+
+        XItemStack.serialize(item, itemStackConfigurationSection);
+        itemConfigurationSection.set("slot", slot);
+        currentItem.getAndIncrement();
+      });
+    }
+
+
+    ConfigurationSection armourConfigurationSection = configurationSection.getConfigurationSection("Armour");
+    if (armourConfigurationSection == null) {
+      armourConfigurationSection = configurationSection.createSection("Armour");
+
+      if (kit.getKitHelmet() != null) {
+        ConfigurationSection helmetConfigurationSection = armourConfigurationSection.createSection("Helmet");
+        XItemStack.serialize(kit.getKitHelmet(), helmetConfigurationSection);
+      }
+
+      if (kit.getKitChestplate() != null) {
+        ConfigurationSection chestplateConfigurationSection = armourConfigurationSection.createSection("Chestplate");
+        XItemStack.serialize(kit.getKitChestplate(), chestplateConfigurationSection);
+      }
+
+      if (kit.getKitLeggings() != null) {
+        ConfigurationSection leggingsConfigurationSection = armourConfigurationSection.createSection("Leggings");
+        XItemStack.serialize(kit.getKitLeggings(), leggingsConfigurationSection);
+      }
+
+      if (kit.getKitBoots() != null) {
+        ConfigurationSection bootsConfigurationSection = armourConfigurationSection.createSection("Boots");
+        XItemStack.serialize(kit.getKitBoots(), bootsConfigurationSection);
+      }
+    }
+  }
+
+  public void loadKitConfig(Kit kit) {
+    return;
+  }
+
+  private void loadKitConfig(String kit_key) {
+    ConfigurationSection configurationSection = kitsConfig.getConfigurationSection("kit." + kit_key);
+
+    if (configurationSection == null) {
+      plugin.getDebugger().debug(Level.SEVERE, "Kit " + kit_key + " does not have any configuration.");
+      plugin.getDebugger().debug(Level.SEVERE, "Kit" + kit_key + " will not be loaded.");
+      return;
+    }
+
+    String kit_name;
+    if (configurationSection.getString("name") == null) {
+      kit_name = kit_key;
+    }
+    else {
+      kit_name = configurationSection.getString("name");
+    }
```

Comment:
```suggestion
    String kit_name = configurationSection.getString("name", kit_key);
```
Simplify and stick to using default if the value `name` is null